tekton/publish.yaml

```diff
@@ -1,157 +1,179 @@
 apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
-  name: publish-tekton-triggers
+  name: publish-triggers-release
 spec:
-  resources:
-    inputs:
-    - name: source
-      type: git
-      targetPath: go/src/github.com/tektoncd/triggers
-    - name: bucket
-      type: storage
-    outputs:
-    - name: bucket
-      type: storage
-    - name: builtControllerImage
-      type: image
-    - name: builtWebhookImage
-      type: image
-    - name: builtEventListenerSinkImage
-      type: image
-    - name: builtCoreInterceptorsImage
-      type: image
-    - name: notification
-      type: cloudEvent
   params:
+  - name: package
+    description: package to release (e.g. github.com/<org>/<project>)
+    default: github.com/tektoncd/triggers
+  - name: images
+    description: List of cmd/* paths to be published as images
+    default: "controller webhook entrypoint nop kubeconfigwriter git-init imagedigestexporter pullrequest-init"
```

Comment:
interesting - curious why are we using a string param separated by spaces instead of array param here?

Reply:
no specific reason. I followed the working example from tekton-pipelines. 🤓 
we try making it in a future update.